opengrok-indexer/src/main/java/org/opengrok/indexer/analysis/Definitions.java

```diff
@@ -60,27 +60,27 @@
     public static class LineTagMap implements Serializable {
 
         private static final long serialVersionUID = 1191703801007779481L;
-        private final Map<String, Set<Tag>> sym_tags; //NOPMD
+        private final Map<String, Set<Tag>> symTags; //NOPMD
```

Comment:
Is this going to work? I think this breaks backwards compatibility as serialized classes contain the names of the fields (at least based on the simple test I did).

Reply:
Good catch, reverting.